### PR TITLE
Magic Camera

### PR DESCRIPTION
**Pull request by 1whatleytay**
Working on cleaning up some movement, player animations and camera work!

### CODE REVIEW

File: Assets/Prefabs/Player/PlayerAnimatorController.controller [hunk 1]
```diff
@@ -0,0 +1,159 @@
+%YAML 1.1
```
Comment (AlekseyPanas) [inline Assets/Prefabs/Player/PlayerAnimatorController.controller:1]: Please make a subfolder for animation files inside this Player folder. Keep only scripts and the actual player prefab at the root. The fbx should also go into a subfolder

**Review by PyroPoro — APPROVED** [review]
looks good. i saw the demo we can merge

**Review by PyroPoro — DISMISSED** [review]

File: Assets/Prefabs/PlayerCamera/CameraManager.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
+using UnityEngine;
+
+
```
Comment (AlekseyPanas) [inline Assets/Prefabs/PlayerCamera/CameraManager.cs:3]: Please add more documentation on how to use these classes. Ideally reading the doc above the class should give me an idea of where this class fits in with the rest. Please also do this for the other camera classes that were added

**Review by AlekseyPanas — CHANGES_REQUESTED** [review]

File: Assets/Prefabs/Player/PlayerAnimatorController.controller (hunk 1 above)
Comment (1whatleytay) [reply Assets/Prefabs/Player/PlayerAnimatorController.controller:1]: Sure

**Review by AlekseyPanas — APPROVED** [review]
Awesome!

**Review by PyroPoro — APPROVED** [review]

**Pull request merged**